tests/v2/validation/upgrade/kubernetes_test.go

```diff
@@ -106,4 +110,23 @@ func (u *UpgradeKubernetesTestSuite) testUpgradeSingleCluster(clusterName, versi
 		u.T().Logf("[%v]: Validating updated cluster's nodepools kubernetes versions", clusterName)
 		validateNodepoolVersions(u.T(), client, updatedCluster, version, !isCheckingCurrentCluster)
 	}
+
+	err = nodestat.IsNodeReady(client, clusterMeta.ID)
+	require.NoError(u.T(), err)
+
+	clusterToken, err := clusters.CheckServiceAccountTokenSecret(client, clusterName)
+	require.NoError(u.T(), err)
+	assert.NotEmpty(u.T(), clusterToken)
+
+	podResults, podErrors := pods.StatusPods(client, clusterMeta.ID)
+	assert.NotEmpty(u.T(), podResults)
+	assert.Empty(u.T(), podErrors)
+
+	if psact == string(RancherPrivileged) || psact == string(RancherRestricted) {
+		err = psadeploy.CheckPSACT(client, clusterName)
+		require.NoError(u.T(), err)
+
+		_, err = psadeploy.CreateNginxDeployment(client, clusterMeta.ID, psact)
```

Comment:
in the create function, can you add the deployment name to the log? Currently reads as `"Deployment successfully created; this is expected for rancher-privileged!"`

Reply:
Resolved.